docs/src/pages/components/togglebutton/examples/ToggleButtonStylePropsExample.tsx

```diff
@@ -0,0 +1,17 @@
+import { Flex, ToggleButton, useTheme } from '@aws-amplify/ui-react';
+
+export const ToggleButtonStylePropsExample = () => {
+  const { tokens } = useTheme();
+  return (
+    <Flex>
+      <ToggleButton color={tokens.colors.orange[60]}>Press me!</ToggleButton>
+      <ToggleButton
+        style={{
+          color: `${tokens.colors.orange[60]}`,
+        }}
```

Comment:
I think this can be shortened to `color={tokens.colors.orange[60]}`

Reply:
I think I provided one more example through the native `style` prop. I probably should remove it in favor of the styling props.